### DISCUSSION TITLE
Helping indie devs collaborate

### DISCUSSION THREAD

**Discussion by JetForMe** [community/community#60420] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

TL; dr: I really would like to be able to show my private code (and wiki pages) to specific individuals, without having to buy a full seat for them. As an indie developer, I don't have a lot of collaboration, but I do occasionally want to ask colleagues, "What do you think of this?"

I have two GH accounts, one personal, and one organizational. I am an independent developer, and I do consulting. I also contribute to open source projects. I have a need for both public and private repos.

I work on a wide variety of projects, and I have a lot of private repos. I pay for my own seat for my personal and my organization, and I pay for another seat on my One Big Project because I recently gained a regular collaborator on it.

But I have a lot of little side projects, and I have occasional need to show some of that to other developers, usually to get advice, or to show them how to do something. They are not regular contributors to my projects, and who they are varies. It doesn't make economic sense for me to pay for seats for each person.

Would GH ever consider a way for me give read-only access to my private repos to specific individuals? Perhaps I could have some limited number of authorized users across all my repos. They would not be allowed to write or fork, just to look around. maybe clone it, to help me diagnose a problem, or try the code out for themselves, although I can work around that easily enough by just sending them a zip file.

Mostly I just want to be able to quickly show someone some code or a wiki page by sending them a link to that page, but I don't want the page generally accessible.
